sdk/python/jobs/pipelines/1i_pipeline_with_spark_nodes/pipeline_with_spark_nodes.ipynb

```diff
@@ -238,9 +238,9 @@
    "description": "Create pipeline with spark node"
   },
   "kernelspec": {
-   "display_name": "Python 3.10 - SDK V2",
+   "display_name": "Python 3.7.10 ('vnext')",
```

Comment:
Let's not have different kernelspec for each different notebook.

Reply:
Reverted kernelspec changes.